docs/ssm-on-aws/ssm-on-aws.md

```diff
@@ -0,0 +1,374 @@
+        "security": "none",
+        "scalability": "medium"
+    },
+
+    "steady-state-hypothesis": {
+        "title": "EC2 instance should respond within 6 seconds",
+        "probes": [
+            {
+                "name": "http-response-time",
+                "type": "probe",
+                "provider": {
+                    "type": "python",
+                    "module": "chaoscloud.probes.http",
+                    "func": "time_http_call",
+                    "arguments": {
+                        "url": "${endpoint_url}"
+                    }
+                },
+                "tolerance": {
+                    "name": "http-response-time-under",
+                    "type": "probe",
+                    "provider": {
+                        "type": "python",
+                        "module": "chaoscloud.tolerances.http",
+                        "func": "response_time_under",
+                        "arguments": {
+                            "duration": 6000
+                        }
+                    }
+                }
+            }
+        ]
+    },
+    "method": [
+        {
+            "type": "action",
+            "name": "inject-cpu",
+            "provider": {
+                "type": "process",
+                "path": "aws",
+                "arguments": "ssm send-command --document-name \"cpu-stress\" --document-version \"1\"
+--targets '[{\"Key\":\"InstanceIds\",\"Values\":[\"${instance_id}\"]}]'
+--parameters '{\"duration\":[\"180\"],\"cpu\":[\"0\"]}' --timeout-seconds 60
+--max-concurrency \"50\" --max-errors \"0\" --output-s3-bucket-name \"gtf-chaos-output\"
+--cloud-watch-output-config '{\"CloudWatchOutputEnabled\":true}'
+--region eu-west-1 --query 'Command.CommandId' > CommandId.info"
+            }
+        }
+    ],
+    "rollbacks": [
+        {
+            "type": "action",
+            "name": "Stop SSM run command",
+            "provider": {
+                "type": "process",
+                "path": "aws",
+                "arguments": "ssm cancel-command --command-id $(cat CommandId.info) "
+            },
+            "pauses": {
+                "after": 10
+            }
+        }
+    ]
+}
+```
+
+!!!Note
+    You will need to add your own S3bucket name.
+
+
+It worth connecting to your EC2 instance now using [SSH][SshAccessToEc2] and running ```htop``` so you can see the impact of the SSM command on the CPU.
+
+The experiment can be run with a  command similar to the following, but you should replace the ```ENDPOINT_URL``` and the ```INSTANCE_ID``` with your own EC2 values:
+
+```bash
+ENDPOINT_URL=http://ec2-52-18-8-61.eu-west-1.compute.amazonaws.com/random/4 \
+INSTANCE_ID=i-086ccae1f2de9b012 \
+chaos run verification-ec2-responds-processor-load-stage.json
+```
+
+Shortly after the experiment begins the ```htop``` session should look similar to:
+
+![htop screenshot][HtopScreen]
+
+At this point, some stress has been applied to the CPU and a Steady
```

Comment:
Could this be collected during the experiment's method as proof? Maybe in a background probe? Maybe a new probe using psutil?

Reply:
Nice idea!